spec/registry.xml

```diff
@@ -0,0 +1,250 @@
+      <!-- The formatting style used for day periods like "in the morning", "am", "noon", "n" etc. -->
+      <option name="dayPeriod" values="narrow short long"/>
+      <!-- Numbering system to use. -->
+      <option name="numberingSystem" values="arab arabext bali beng deva fullwide gujr guru hanidec khmr knda laoo latn limb mlym mong mymr orya tamldec telu thai tibt"/>
+      <!-- The time zone to use. The only value implementations must recognize
+         is "UTC"; the default is the runtime's default time zone.
+         Implementations may also recognize the time zone names of the IANA time zone
+         database, such as "Asia/Shanghai", "Asia/Kolkata", "America/New_York".
+      -->
+      <option name="timeZone" pattern="timeZoneId"/>
+      <!-- The hour cycle to use. -->
+      <option name="hourCycle" values="h11 h12 h23 h24"/>
+      <!-- The representation of the weekday. -->
+      <option name="weekday" values="long short narrow"/>
+      <!-- The representation of the era. -->
+      <option name="era" values="long short narrow"/>
+      <!-- The representation of the year. -->
+      <option name="year" values="numeric 2-digit"/>
+      <!-- The representation of the month. -->
+      <option name="month" values="numeric 2-digit long short narrow"/>
+      <!-- The representation of the day. -->
+      <option name="day" values="numeric 2-digit"/>
+      <!-- The representation of the hour. -->
+      <option name="hour" values="numeric 2-digit"/>
+      <!-- The representation of the minute. -->
+      <option name="minute" values="numeric 2-digit"/>
+      <!-- The representation of the second. -->
+      <option name="second" values="numeric 2-digit"/>
+      <!-- The number of digits used to represent fractions of a second
+        (any additional digits are truncated). -->
+      <option name="fractionalSecondDigits" values="1 2 3"/>
+      <!-- The localized representation of the time zone name. -->
+      <option name="timeZoneName" values="long short shortOffset longOffset shortGeneric longGeneric"/>
+    </formatSignature>
+
+    <!-- TODO: IGNORED for now. To discuss. -->
+    <!-- The locale matching algorithm to use. -->
+    <!--
+    <option name="localeMatcher" values="lookup best-fit" default="best-fit"/>
+    -->
+    <!-- The format matching algorithm to use. -->
+    <!--
+    <option name="formatMatcher" values="basic best-fit" default="best-fit"/>
+    -->
+    <!-- Whether to use 12-hour time (as opposed to 24-hour time). -->
+    <!-- I think it overlaps with "hourCycle"?
+    <option name="hour12" values="true false"/>
+    -->
+
+  </function>
+
+  <function name="number">
+    <!-- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat -->
+    <description>Language-sensitive number formatting</description>
```

Comment:
s/Language-sensitive/Locale-sensitive/g (won't repeat this later)

Reply:
Done